metadata-ingestion/tests/unit/test_bigquery_source.py

```diff
@@ -330,8 +330,10 @@ def test_table_processing_logic(client_mock, data_dictionary_mock):
 
     source = BigqueryV2Source(config=config, ctx=PipelineContext(run_id="test"))
 
-    _ = source.get_tables_for_dataset(
-        conn=client_mock, project_id="test-project", dataset_name="test-dataset"
+    list(
```

Comment:
make it clear that we're throwing away the value

```suggestion
    _ = list(
```